<ui version="4.0" >
 <class>BoundaryConditionsVelocityInletForm</class>
 <widget class="QWidget" name="BoundaryConditionsVelocityInletForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBoxVelocity" >
     <property name="title" >
      <string>Velocity</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" >
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>16</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" >
          <item>
           <widget class="QComboBox" name="comboBoxVelocity" />
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditVelocity" />
          </item>
          <item>
           <widget class="QLabel" name="labelUnitVelocity" >
            <property name="text" >
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>17</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonVelocityFormula" >
            <property name="text" >
             <string/>
            </property>
            <property name="icon" >
             <iconset resource="resources_pages.qrc" >
              <normaloff>:/new/prefix1/icons/22x22/edit-find-replace.png</normaloff>:/new/prefix1/icons/22x22/edit-find-replace.png</iconset>
            </property>
            <property name="iconSize" >
             <size>
              <width>22</width>
              <height>22</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>22</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox" >
        <property name="title" >
         <string>Direction</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignCenter</set>
        </property>
        <property name="flat" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" >
        <item>
         <spacer name="horizontalSpacer_2" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" >
          <item>
           <widget class="QComboBox" name="comboBoxDirection" />
          </item>
          <item>
           <spacer name="horizontalSpacer_3" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>13</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonDirectionFormula" >
            <property name="text" >
             <string/>
            </property>
            <property name="icon" >
             <iconset resource="resources_pages.qrc" >
              <normaloff>:/new/prefix1/icons/22x22/edit-find-replace.png</normaloff>:/new/prefix1/icons/22x22/edit-find-replace.png</iconset>
            </property>
            <property name="iconSize" >
             <size>
              <width>22</width>
              <height>22</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_4" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="frameDirectionCoordinates" >
        <property name="minimumSize" >
         <size>
          <width>16</width>
          <height>42</height>
         </size>
        </property>
        <property name="frameShape" >
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth" >
         <number>0</number>
        </property>
        <layout class="QHBoxLayout" name="_4" >
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>21</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="_5" >
           <item>
            <widget class="QLabel" name="labelXVelocity" >
             <property name="text" >
              <string>X</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditDirectionX" >
             <property name="maximumSize" >
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="labelYVelocity" >
             <property name="text" >
              <string>Y</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditDirectionY" >
             <property name="maximumSize" >
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="labelZVelocity" >
             <property name="text" >
              <string>Z</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditDirectionZ" >
             <property name="maximumSize" >
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources_pages.qrc" />
 </resources>
 <connections/>
</ui>
